ppa-update-tools/checkbox-ppas-for-canary.yaml

```diff
@@ -0,0 +1,58 @@
+channel: edge
+required-packages:
+ - source: checkbox-ng
+   package: checkbox-ng
+   versions: [ "18.04", "20.04", "22.04", "23.04", "23.10", "24.04" ]
+   architectures: [ all ]
+   
+ - source: checkbox-provider-base
+   package: checkbox-provider-base
+   versions: [ "20.04", "22.04", "23.04", "23.10", "24.04" ]
+   architectures: [ amd64, arm64, armhf, ppc64el, riscv64, s390x ]
+   # The package is not built in 18.04 for riscv64
+   include: 
+      versions: [ "18.04"]
+      architectures: [ amd64, arm64, armhf, ppc64el, s390x ]
```

Comment:
What I meant by "just including" is this:
```
 - source: checkbox-provider-base
   package: checkbox-provider-base
   versions: [ "20.04", "22.04", "23.04", "23.10", "24.04" ]
   architectures: [ amd64, arm64, armhf, ppc64el, riscv64, s390x ]
 - source: checkbox-provider-base
   # Launchpad doesn't build riscv64 packages for releases 18.04 and older,
   # so we need to define it explicitly for non-riscv64 archs.
   package: checkbox-provider-base
      versions: [ "18.04"]
      architectures: [ amd64, arm64, armhf, ppc64el, s390x ]
```

I don't see the point of introducing anything new to handle this.

Reply:
You are right there, I overthink it. I will just have it as you proposed here.